### PR TITLE
Add a few more things to firedrills.

### PR DESCRIPTION
**Pull request by r0mant**
According to the topics a customer expressed interest in:

* How Planet container is configured.
* How to increase logging of certain components.
* How to collect only Kubernetes logs.

### CODE REVIEW

File: firedrills.md [hunk 1]
```diff
@@ -31,6 +31,30 @@ node-1$ sudo systemctl status kube-apiserver
    Active: inactive (dead)
 ```
 
+#### Planet Container Properties
+
+Planet is a runc-based container that unshares the following namespaces:
```
Comment (a-palchikov) [inline firedrills.md]: nit: `unshares` comes as a bit of too technical to me. I'd use something like `creates private copies of the following namespaces so it is able to manage resources independently of other processes`

File: firedrills.md [hunk 2]
```diff
@@ -168,6 +192,40 @@ node-1-planet$ journalctl -u kube-apiserver --no-pager
...
+node-1-planet$ systemctl status kube-kubelet
+● kube-kubelet.service - Kubernetes Kubelet
+   Loaded: loaded (/lib/systemd/system/kube-kubelet.service; static; vendor preset: enabled)
+   Active: active (running) since Mon 2019-10-07 19:19:13 UTC; 3 months 9 days ago
+```
+
+Once the unit's service spec file is identified, edit it and add a `-v5` flag somewhere to the `ExecStart` command:
+
+```bash
+node-1-planet$ nano /lib/systemd/system/kube-kubelet.service
```
Comment (a-palchikov) [inline firedrills.md]: I think the proper way of making changes to systemd units is [`systemctl edit`](https://www.freedesktop.org/software/systemd/man/systemctl.html#edit%20UNIT%E2%80%A6) - i.e. `systemctl edit --full service` which will create drop-ins in the corresponding directory and auto-reload after making changes.

File: firedrills.md [hunk 3]
```diff
@@ -168,6 +192,40 @@ node-1-planet$ journalctl -u kube-apiserver --no-pager
...
+```
+
+Finally, we can restart kubelet which will start producing more debug output in its logs:
+
+```bash
+node-1-planet$ systemctl restart kube-kubelet
+node-1-planet$ journalctl -u kube-kubelet -f
+```
+
+Note that the changes we've made to the systemd unit will persist across the Planet container restarts but will be reset when a new version of Planet is rolled during system upgrade, for example.
```
Comment (knisbet) [inline firedrills.md]: > Note that the changes we've made to the systemd unit will persist across the Planet container restarts but will be reset when a new version of Planet is rolled during system upgrade, for example.

Maybe something more along the lines of:

Note that the changes we've made to the systemd unit will persist across Planet container restarts but will be lost during a gravity upgrade that includes a new version of planet due to the inclusion of a new rootfs.

File: firedrills.md (hunk 2 above)
Comment (r0mant) [reply firedrills.md]: Added this too.

**Pull request merged**